app/client/components/Layout.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
 import { Box } from '@rebass/grid'
-import Centerer from './Centerer'
-import AppBar from '../AppBar'
+import Centerer from '@elifesciences/component-elife-ui/client/global/layout/Centerer'
+import WrappedAppBar from './WrappedAppBar'
```

Comment:
Should this be `WrappedTopBar` ?

Reply:
TopBar is just the styled 72px box with a bottom grey border. AppBar includes the profile menu and the navigation menu wrapped inside this styling and WrappedAppBar is just that but with the GQL queries for fetching user details.

Topbar gets used on the redirect page without any of the profile or navigation stuff so they're different but require to same TopBar wrapper.

Reply:
Maybe something for the future would be to come up with a naming convention that would indicate "thing with GQL" / "Just a styled Thing" etc

Reply:
Plan for the future is to use HOC's for graphql stuff so you'd just wrap the page in a `withGQL(Layout)` wrapper 💯